networking.md

```diff
@@ -250,113 +250,75 @@ spec:
 
 ### Configure exposing services to isolated networks
 #### Service API component
-Each API component of a service supports exposing their endpoint to an isolated network
+Each API component of a service supports service overrides which allows to customize the created k8s service for an endpoint. The default service type for an endpoint is `ClusterIP` service. To expose an API endpoint to the isolated network a `LoadBalancer` can be used.
 
-```yaml
-  externalEndpoints:
-    description: ExternalEndpoints, expose a VIP using a pre-created IPAddressPool
-    items:
-      description: MetalLBConfig to configure the MetalLB loadbalancer service
-      properties:
-        endpoint:
-          description: Endpoint, OpenStack endpoint this service maps to
-          enum:
-          - internal
-          - public
-          type: string
-        ipAddressPool:
-          description: IPAddressPool expose VIP via MetalLB on the IPAddressPool
-          minLength: 1
-          type: string
-        loadBalancerIPs:
-          description: LoadBalancerIPs, request given IPs from the pool
-            if available. Using a list to allow dual stack (IPv4/IPv6) support
-          items:
-            type: string
-          type: array
-        sharedIP:
-          default: true
-          description: SharedIP if true, VIP/VIPs get shared with multiple services
-          type: boolean
-        sharedIPKey:
-          default: ""
-          description: SharedIPKey specifies the sharing key which gets
-            set as the annotation on the LoadBalancer service. Services
-            which share the same VIP must have the same SharedIPKey. Defaults
-            to the IPAddressPool if SharedIP is true, but no SharedIPKey
-            specified.
-          type: string
-      required:
-      - endpoint
-      - ipAddressPool
-```
+For services which get registered to different OpenStack endpoint types a hash `map[service.Endpoint]service.RoutedOverrideSpec` is used, where the key is either `public` or `internal`.
 
-Example:
+Example: To expose the internal keystone endpoint using a MetalLB k8s LoadBalancer via the service override the `LoadBalancer` type in the spec and required annotations in the metadata has to be set:
```

Comment:
I would put a comma after `LoadBalancer via the service override`.  It helps the reader parse the long sentence.

Reply:
done